packages/contracts/dump_localhost.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+set -e
+STATE_FILE="anvil_state.json"
+
+# start anvil in the background, with --dump-state
+anvil --dump-state $STATE_FILE &
+PID=$!
+
+# wait for anvil to start
+sleep 1
+
+# remove previous deployment
+rm -rf deployments/localhost
+rm -rf export/abi/localhost.json
+
+# deploy project to localhost (anvil)
+npx hardhat deploy --network localhost --export export/abi/localhost.json
```

Comment:
Does foundry have some tool to make this deployment?

The same as before, is it possible to use `cartesi/rollups-cli` for this?

Reply:
No, deploy through `forge` is still behind `hardhat`